engine/table/src/test/java/io/deephaven/engine/table/impl/QueryTableAggregationTest.java

```diff
@@ -149,7 +149,7 @@ public void testStaticNoKeyByWithChunks() {
     @Test
     public void testStaticReinterpretableKeyByWithChunks() {
         final String nowName = "__now_" + Thread.currentThread().hashCode() + "__";
-        QueryScope.addParam(nowName, DateTime.now());
+        QueryScope.addParam(nowName, DateTime.nowMillis());
```

Comment:
Let's change these two back. Other cases in other files, too.

Reply:
Done